solidity/contracts/utility/Math.sol

```diff
@@ -56,8 +56,10 @@ library Math {
         uint256 _d,
         uint256 _max
     ) internal pure returns (uint256, uint256) {
-        if (_n > _max || _d > _max) return normalizedRatio(_n, _d, _max);
-        return (_n, _d);
+        (uint256 n, uint256 d) = (_n, _d);
+        if (n > _max || d > _max) (n, d) = normalizedRatio(n, d, _max);
```

Comment:
1. what is this logical change?
2. missing curly brackets etc.

Reply:
1. This is just used as a preliminary step to the logical change in this function, of returning the smallest possible result when the numerator and the denominator are equal (i.e., `1, 1`).

2. Was also missing before. I wanted to keep it that way in order to minimize visible changes during code review.